### DISCUSSION TITLE
Totally disappointed about ( RTL )

### DISCUSSION THREAD

**Discussion by RabbiIslamRony** [mdbootstrap/TW-Elements#2073] [Share ideas & request features]
I am thoroughly disappointed with the lack of **RTL (right-to-left) support**. Although this framework boasts many features, it falls short in supporting RTL, which most other frameworks do. Could you please take steps to implement **RTL support?**

**Comment by devShamim** [comment]
I am dissapointed too, but I hope they will take necessary steps soon.

**Comment by juujisai** [comment]
Hi! Thanks for the feedback. At this moment we are working on bug fixes and other important tasks (unit testing, layout updates etc). We will work on RTL / tutorials how to integrate RTL with TW Elements in future releases. I've added a new issue for that: https://github.com/mdbootstrap/TW-Elements/issues/2077

**Reply by RabbiIslamRony** [reply]
Thank you for your prompt response. I understand the importance of prioritizing tasks such as bug fixes and updates. However, the absence of RTL support significantly hinders our ability to utilize Tailwind Elements in projects that require such functionality. This feature is crucial for us to create standards-compliant products efficiently. I appreciate that issue #2077 has been opened to address this, and I am looking forward to the implementation of RTL support in the near future. Thank you for considering the needs of your diverse user base 😔

**Reply by juujisai** [reply]
I think, right now, most of the rtl problems can be resolved via class customization but there are a lots of components that would require a lot of work. Nevertheless, we will take a look at that and try to find the best solution.

I'll leave the example on how to use class customization if you were to try it:
https://tw-elements.com/docs/standard/forms/datepicker/#docsTabsAPI - section Classes
